alica_tests/src/DistBallRobot.cpp

```diff
@@ -35,10 +35,11 @@ UtilityInterval DistBallRobot::eval(IAssignment ass, const Assignment* oldAss, c
             pos = 1;
         }
 
+        auto* worldModel = dynamic_cast<const alicaTests::TestWorldModel*>(wm);
         if (this->robotId == agentID8) {
-            curPosition = alicaTests::TestWorldModel::getOne()->robotsXPos[pos];
+            curPosition = worldModel->robotsXPos[pos];
         } else {
-            curPosition = alicaTests::TestWorldModel::getTwo()->robotsXPos[pos];
+            curPosition = worldModel->robotsXPos[pos];
```

Comment:
this won't work because you need to access the world model of the second (other) agent

Reply:
It seems like both DistBallRobot and DistXContourTest are not used anymore and we can also get rid of them.